### PR TITLE
Remove classification

### PR DESCRIPTION
**Pull request by michael-bryson**
Classification has been removed from the data processing service and all utilities. This includes the database container which only installs the workflow and boilerplate databases now. The documentation has also been updated and can be viewed on the dev build here:
https://pages.github.houston.softwaregrp.net/sepg-docs-qa/CAFDataProcessing_CI_data-processing-service_remove_classification/

The dev build itself is passing and can be seen here:
http://sou-jenkins2.hpeswlab.net:8080/job/CAFDataProcessing/view/Developer/job/CAFDataProcessing~data-processing-service~remove_classification~CI/

### CODE REVIEW

**Review by dermot-hardy — APPROVED** [review]
Looks OK to me but needs to be approved by @mmaBel.

File: utils/data-processing-initialization/src/main/java/com/github/cafdataprocessing/utilities/initialization/WorkflowInitializer.java [hunk 1]
```diff
@@ -54,19 +54,15 @@
...
      * @param boilerplateNameResolver for use in resolving expression/tag names to IDs in workflow.
      * @param actionTypeNameResolver for use in resolving action type names to IDs in workflow.
-     * @param classificationWorkflowNameResolver for use in resolving classification workflow names to IDs in processing
-     *                                           workflow.
      */
     public WorkflowInitializer(String processingApiUrl, BoilerplateNameResolver boilerplateNameResolver,
-                               ActionTypeNameResolver actionTypeNameResolver,
-                               ClassificationWorkflowNameResolver classificationWorkflowNameResolver){
+                               ActionTypeNameResolver actionTypeNameResolver){
         final ApiClient apiClient = new ApiClient();
```
Comment (mmaBel) [inline utils/data-processing-initialization/src/main/java/com/github/cafdataprocessing/utilities/initialization/WorkflowInitializer.java]: This is a breaking change for Darwin integration point. We would need to increase the version number for this project accordingly.
https://github.houston.softwaregrp.net/Verity/common-lib/blob/b47139d4380813a48888f00c3410b74d23320dc1/aspen-common/policy-common/src/main/java/com/hpe/aspen/policy/common/PolicyWorkflowService.java#L108

**Review by mmaBel — CHANGES_REQUESTED** [review]

File: utils/data-processing-initialization/src/main/java/com/github/cafdataprocessing/utilities/initialization/WorkflowInitializer.java (hunk 1 above)
Comment (dermot-hardy) [reply utils/data-processing-initialization/src/main/java/com/github/cafdataprocessing/utilities/initialization/WorkflowInitializer.java]: Good catch.  Let's take on the work to update the code that they have that uses this as well.

**Comment by michael-bryson** [comment]
Version has been reverted back to 1.4.0 and initialization project moved into it own repo here:
https://github.com/CAFDataProcessing/data-processing-initialization

**Comment by mmaBel** [comment]
I think the release notes should be updated to say the workflow initializer has been moved to the new repo and a link to that repository. Aside from that looks good.

**Review by mmaBel — APPROVED** [review]

**Comment by michael-bryson** [comment]
Dev build is failing as it cannot currently find the initialization library at 2.0.0-SNAPSHOT:
http://sou-jenkins2.hpeswlab.net/job/CAFDataProcessing/view/Developer/job/CAFDataProcessing~data-processing-service~remove_classification~CI/7/console

**Comment by mmaBel** [comment]
Subsequent build passed. An issue in the with the artifacts proxy syncing delay perhaps?

http://sou-jenkins2.hpeswlab.net/job/CAFDataProcessing/view/Developer/job/CAFDataProcessing~data-processing-service~remove_classification~CI/8/

**Pull request merged**